lib/osf-components/addon/components/osf-layout/registries-side-nav/x-link/component.ts

```diff
@@ -24,6 +29,11 @@ export default class XLink extends Component {
 
     onClick?: () => void;
 
+    @not('media.isDesktop') showMobileView!: boolean;
+
+    @and('isDrawer', 'showMobileView')
+    isInert!: boolean;
+
```

Comment:
So `showMobileView` is whenever the window is not desktop size. When will `isDrawer` be true then?